lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -1,236 +0,0 @@
-#!/usr/bin/python
```

Comment:
@shitalmule04 Any reason that you removed this entire file ?

Reply:
I still not made any changes in lib/ansible/modules/cloud/amazon/_ec2_ami_search.py, it added  my mistake. After that I tried to rebase branch so, this file is removed. Is it ok, if I commit cli/config.py and  amazon/ec2_ami_copy.py  at same time?